community/planning/griddle/openapi.yaml

```diff
@@ -0,0 +1,2146 @@
+            evaluated. After the server receives the versions for validation, it
+            will attempt to execute the transactions contained in the batch list.
+            If each of the transactions are successful, the distributed ledger's
+            state will be updated and Grid will receive state delta events from the
+            distributed ledger. The Grid Daemon processes these events and updates
+            the Grid database accordingly.
+            This means that there is a delay between when transactions are
+            submitted and when the new data is actually available to fetch using
+            the Grid REST API GET endpoints. This delay can range from a few
+            milliseconds to several minutes depending on the underlying ledger.
+            Developers should take this delay into account when designing apps
+            on top of Grid.
+          content:
+            application/json:
+              schema:
+                properties:
+                  link:
+                    $ref: "#/components/schemas/BatchPostResponse"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "500":
+          $ref: "#/components/responses/500ServerError"
+        "503":
+          $ref: "#/components/responses/503ServiceUnavailable"
+  /purchase-orders/{uid}/versions/{version_id}:
+    get:
+      tags:
+        - Purchase Orders
+      summary: Fetches a single purchase order version with the given ID
+      operationId: get_purchase_order_version
+      parameters:
+        - name: version_id
+          in: path
+          description: ID of the version to fetch
+          required: true
+          schema:
+            type: string
+        - name: uid
+          in: path
+          description: ID of the PO to fetch
+          required: true
+          schema:
+            type: string
+        - $ref: "#/components/parameters/service_id"
+        - $ref: "#/components/parameters/page_offset"
+        - $ref: "#/components/parameters/page_limit"
+      responses:
+        "200":
+          description: |
+            Successful request. The response will include a JSON object
+            representing the purchase order version.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PurchaseOrderVersionList"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "404":
+          $ref: "#/components/responses/404NotFound"
+        "500":
+          $ref: "#/components/responses/500ServerError"
+        "503":
+          $ref: "#/components/responses/503ServiceUnavailable"
+    put:
+      tags:
+        - Purchase Orders
+      summary: Updates a single purchase order version with the given ID
+      operationId: put_purchase_order_version
+      parameters:
+        - name: version_id
+          in: path
+          description: ID of the version to fetch
```

Comment:
Could maybe be more descriptive, like `ID of the version to be updated` and the po_uid could be described as `UID of the purchase order this version belongs to`